scene/resources/packed_scene.cpp

```diff
@@ -380,7 +380,27 @@ Node *SceneState::instantiate(GenEditState p_edit_state) const {
 				//if node was not part of instance, must set its name, parenthood and ownership
 				if (i > 0) {
 					if (parent) {
-						parent->_add_child_nocheck(node, snames[n.name]);
+						Node *existing = nullptr;
+						if (Engine::get_singleton()->is_editor_hint()) {
+							existing = parent->get_node_or_null((String)snames[n.name]);
+						}
+						if (existing) {
+							// There's already a node in the same parent with the same name.
+							// This means that somehow the node was added both to the scene being
+							// loaded and another one instantiated in the former, maybe because of
+							// manual edition, or a bug in scene saving, or a loophole in the workflow
```

Comment:
```suggestion
							// manual editing, or a bug in scene saving, or a loophole in the workflow
```

Not which is (more) correct